Tests/TestArgumentsHistory/ArgumentsHistoryTests.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+class ArgumentsHistoryTests: MockoloTestCase {
+    
```

Comment:
Can you add a generic function test case?

Reply:
Added in a26573b and b35d242.